addons/default/visiosoft/profile-module/src/ProfileModuleServiceProvider.php

```diff
@@ -139,6 +143,8 @@ class ProfileModuleServiceProvider extends AddonServiceProvider
 
         // CacheController
         'ajax/get-user-info' => 'Visiosoft\ProfileModule\Http\Controller\CacheController@getUserInfo',
+	    'ajax/setEducation' => 'Visiosoft\ProfileModule\Http\Controller\MyProfileController@setEducation',
+	    'ajax/getEducation' => 'Visiosoft\ProfileModule\Http\Controller\MyProfileController@getEducation',
```

Comment:
you can put these routes under // MyProfileController, and replace "ajax" with "api"